Resources/Prototypes/_Goobstation/Recipes/Lathes/medical.yml

```
@@ -6,3 +6,26 @@
+    Plasma: 100
+
+- type: latheRecipe
+  id: RapidSyringeGun
+  result: RapidSyringeGun
+  completetime: 5
+  materials:
+    Steel: 1000
+    Glass: 1000
+    Plastic: 1000
+    Gold: 200
+    Plasma: 200
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)